### DISCUSSION TITLE
Example using Net7.0

### DISCUSSION THREAD

**Discussion by seanrobertwright** [elsa-workflows/elsa-core#3972] [Q&A]
Hi, I was wondering if there is an example that uses the Net7.0 framework.  All the examples use the Startup class (Startup.cs) but that was done away with Net6.0.

Thanks in advance,

Sean

**Comment by sfmskywalker** [comment]
For Elsa 2, I don't think we have any samples yet, but it makes sense to update the existing ones.
For Elsa 3, all samples target .NET 7. Perhaps if you take a look at that, you can see how to adapt Elsa 2 samples. Ultimately, you only need the `IServiceProvider` to register the services and the `IApplicationBuilder` to setup the HTTP pipeline.

But, like I said, we should also update the Elsa 2 samples. Please feel free to open an issue for that if you want.

**Comment by mohdali** [comment]
I have the below example which is equivalent to this tutorial I think.

https://v2.elsaworkflows.io/docs/quickstarts/elsa-dashboard-and-server

Startup class will still work though. It is remove from the new templates to simplify things and promote minimal approach.

```cs
using Elsa;
using Elsa.Persistence.EntityFramework.Core.Extensions;
using Elsa.Persistence.EntityFramework.Sqlite;

var builder = WebApplication.CreateBuilder(args);

var services = builder.Services;
var Configuration = builder.Configuration;
var Environment = builder.Environment;

var elsaSection = Configuration.GetSection("Elsa");

// Elsa services.
services
    .AddElsa(elsa => elsa
        .UseEntityFrameworkPersistence(ef => ef.UseSqlite())
        .AddConsoleActivities()
        .AddHttpActivities(elsaSection.GetSection("Server").Bind)
        .AddQuartzTemporalActivities()
        .AddWorkflowsFrom<Startup>()
    );

// Elsa API endpoints.
services.AddElsaApiEndpoints();

// For Dashboard.
services.AddRazorPages();

var app = builder.Build();

if (Environment.IsDevelopment())
{
    app.UseDeveloperExceptionPage();
}



app
    .UseStaticFiles()
    .UseHttpActivities()
    .UseRouting()
    .UseEndpoints(endpoints =>
    {
        // Elsa API Endpoints are implemented as regular ASP.NET Core API controllers.
        endpoints.MapControllers();

        // For Dashboard.
        endpoints.MapFallbackToPage("/_Host");
    });

app.Run();
```
